api/handlers/event.go

```diff
@@ -68,16 +68,17 @@ func PostEventHandlerFunc(params event.PostEventParams, principal *models.Princi
 // GetEventHandlerFunc returns an event specified by keptnContext and eventType
 func GetEventHandlerFunc(params event.GetEventParams, principal *models.Principal) middleware.Responder {
 
-	logger := keptnutils.NewLogger(*params.KeptnContext, "", "api")
+	keptnContext := uuid.New().String()
```

Comment:
Do we really need a new `keptnContext` only for the logging? I do not think so.

Reply:
Reverted the change to use the provided KeptnContext for logging.